arm-insights/2015-04-01/swagger/insightsClient.json

```diff
@@ -0,0 +1,1123 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "InsightsClient",
+    "version": "2015-04-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/providers/microsoft.insights/eventtypes/management/eventcategories": {
+      "get": {
+        "tags": [
+          "Event"
+        ],
+        "operationId": "EventCategories_List",
+        "description": "The list of event categories.",
+        "parameters": [
+          {
+            "name": "api-version",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "default": "2015-04-01",
```

Comment:
If the client does not explicitly provide an API version, then what will the service assume? As the swagger is currently defined, the expectation is that, unless the client sent the API version, the service will *always* serve up data corresponding to the 2015-04-01 schema. regardless of if a new API version has been released. I strongly suspect that this is not what you intend to describe.